apps/bond/components/BondingProducts/Bonding/BondingList/useBondingList.jsx

```diff
@@ -329,12 +330,13 @@ const getLpTokenNamesForProducts = async (productList, events) => {
 
   return productList.map((component, index) => {
     const { name, poolId, lpChainId, guide } = lpTokenDetailsList[index];
-    const lpTokenLink = getLpTokenLink({
+    const lpLink = getLpLink({
       lpDex: lpTokenDetailsList[index].dex,
       lpChainId,
       lpPoolId: poolId,
       lpAddress: component.token,
     });
+    const lpTokenLink = `https://etherscan.io/token/${component.token}`;
```

Comment:
this is the link to the lp token representation on ethereum (often a bridged token)